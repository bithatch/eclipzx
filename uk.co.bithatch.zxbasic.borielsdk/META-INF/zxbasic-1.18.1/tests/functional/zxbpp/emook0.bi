#define SetREG(reg,val) \
        LD A,reg \
        LD BC,val

ASM
        SetREG(A,254)
end Asm
