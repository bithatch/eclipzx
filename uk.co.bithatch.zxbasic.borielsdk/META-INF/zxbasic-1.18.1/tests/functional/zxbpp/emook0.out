#line 1 "emook0.bi"


ASM
        
        LD A,A 
        LD BC,254
#line 7
end Asm
